sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -34,12 +34,21 @@ protected ConversationAnalysisClient()
         {
         }
 
-        /// <summary> [Protocol Method] Analyzes the input conversation utterance. </summary>
-        /// <param name="content"> The content to send as the body of the request. Details of the request body schema are in the Remarks section below. </param>
+        /// <summary>
+        /// [Protocol Method] Analyzes the input conversation utterance.
+        /// <list type="bullet">
+        /// <item>
+        /// <description>
+        /// This <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md">protocol method</see> allows explicit creation of the request and processing of the response for advanced scenarios.
+        /// </description>
+        /// </item>
+        /// </list>
```

Comment:
Nit: compared to .NET APIs, this feels more like it should be in `<remarks>`, but it's an improvement nonetheless.